Tasks/Common/ios-signing-common/ios-signing-common.ts

```diff
@@ -671,8 +678,12 @@ async function printFromPlist(itemToPrint: string, plistPath: string) {
     return printedValue;
 }
 
-function getProvisioningProfilePath(uuid: string): string {
-    return tl.resolve(getUserProvisioningProfilesPath(), uuid.trim().concat('.mobileprovision'));
+function getProvisioningProfilePath(uuid: string, provProfilePath?: string): string {
+    let profileExtension: string;
```

Comment:
Assign a default value to avoid "undefined" being appended below, or simply the code by making provProfilePath a non-nullable arg.